tests/test_interop_scenarios.rs

```diff
@@ -0,0 +1,263 @@
+use openmls::prelude::*;
+use std::convert::TryFrom;
+use test_macros::ctest;
+
+mod utils;
+
+use utils::managed_utils::*;
+
+// The following tests are test corresponding the the interop test scenarios
```

Comment:
" tests are test"

Reply:
Good point. I've rewritten that paragraph a bit.